ehcache-impl/src/main/java/org/ehcache/impl/internal/executor/PooledExecutionService.java

```diff
@@ -106,7 +105,7 @@ public void start(ServiceProvider<Service> serviceProvider) {
         throw new IllegalStateException("Pool for default pool alias is null");
       }
     } else {
-      LOGGER.warn("No default pool configured, services requiring thread pools must be configured explicitly using named thread pools");
+      LOGGER.warn("No default pool configured, services requiring thread pools must be configured explicitly using named thread pools. This may result in subsequent failure ahead.");
```

Comment:
There's some redundancy in this new sentence ('subsequent' and 'ahead'). Perhaps we could consider folding this statement in with the second clause in the previous sentence:
```java
LOGGER.warn("No default pool configured. Services requiring thread pools will fail if not explicitly configured to use a named pool.");
```